food.js

```diff
@@ -0,0 +1,131 @@
+// Your first task is to build a product page that displays all brands of dog food, the different types, with the price and size for each container volume.
+
+
+//DOM getter variable
+var dogContentDiv = document.getElementById("dogContent");
+var catContentDiv = document.getElementById("catContent");
+
+//I used the 'append child' method to CREATE a div for each loop-through, just in case. It seems like better practice for future functionality. Declared out here so I can use them for both cats and dogs.
+var makeMasterdogDiv = function() {
+  let masterDiv = document.createElement("div");
+  masterDiv.className = "masterdiv col-xs-6";
+  dogContentDiv.appendChild(masterDiv);
+  return masterDiv;
+}
+
+var makeMastercatDiv = function() {
+  let masterDiv = document.createElement("div");
+  masterDiv.className = "masterdiv col-xs-6 cat";
+  catContentDiv.appendChild(masterDiv);
+  return masterDiv;
+}
+var makeTypeDiv = function() {
+  let typeDiv = document.createElement("div");
+  typeDiv.className = "typeDiv";
+  masterDiv.appendChild(typeDiv);
+  return typeDiv;
+}
+//div for breeds, only used during cat loop. to make all the breeds appear in a row.
+var makeBreedDiv = function() {
+  let breedDiv = document.createElement("p");
+  breedDiv.className = "breedDiv";
+  masterDiv.appendChild(breedDiv);
+  breedDiv.innerHTML = `ideal for breeds: `
+  return breedDiv;
+}
+
+var makeVolumeDiv = function() {
+  let volumeDiv = document.createElement("p");
+  volumeDiv.className = "volumeDiv";
+  typeDiv.appendChild(volumeDiv);
+  return volumeDiv;
+} 
+//====================
+//function that runs when DOG json is loaded: a series of nested forEach loops. Probably rather a time-consuming method since each loop and subloop and sub-subloop drills top-down through the json each time. 
+//====================
+var createdogcard = function() {
+  console.log("dog food loaded");
+  //parse dog food object.
+  var dogFood = JSON.parse(dogFoodRequest.responseText);
+  console.log("dog food object", dogFood );
+  
+  //loop one: create master div with brand title.
+  dogFood.dog_brands.forEach(function(brand, brandindex){
+    masterDiv = makeMasterdogDiv();
+    console.log("brand of dog food", dogFood.dog_brands[brandindex].name );
+    masterDiv.innerHTML = `<h4 class = "brandname">${dogFood.dog_brands[brandindex].name}</h4>`;
```

Comment:
What you're doing here is neglecting the awesomeness that is the ```forEach``` loop.  Within it as defined ```brand``` _is_ ```dogFood.dog_brands[brandindex]```, and as such, you can just hit it up as ```brand.name``` instead of having to use the index upon the entire object chain.